File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/java/chunk_D/f16p2_V3_boxedreturn.java
import java.util.Map;

class Main {
    static Object mapTotal() {
        // BEGIN PROBE F16.P2
        var mp = Map.of("a", 1);
        int total = 0;
        for (var e : mp.entrySet()) total += e.getValue();
        int miss = mp.getOrDefault("b", 0);
        return total + miss;
        // END PROBE F16.P2
    }

    public static void main(String[] args) {
        Object r = mapTotal();
        System.out.println(r.getClass().getName() + " " + r);
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/kotlin/chunk_D/f16p2_V3_boxedreturn.kt
fun probe(): Any {
    // BEGIN PROBE F16.P2
    val mp = mapOf("a" to 1)
    val total = mp.entries.sumOf { it.value }
    val miss = mp["b"] ?: 0
    return total + miss
    // END PROBE F16.P2
}
fun main() { val r = probe(); println(r.javaClass.toString() + " " + r) }
